content/docs/use-cases/data-registries.md

```diff
@@ -2,22 +2,21 @@
 
 One of the main uses of <abbr>DVC repositories</abbr> is the
 [versioning of data and model files](/doc/use-cases/data-and-model-files-versioning),
-with commands such as `dvc add`. With the aim to enable reusability of these
-<abbr>data artifacts</abbr> between different projects, DVC also provides
-commands like `dvc import` and `dvc get`. This means that you projects can
-depend on data from other DVC repositories, **similar to a package management
-systems for data science**.
+with tools like the `dvc add` command and `dvc.yaml` files. DVC also enables
```

Comment:
let's just avoid mentioning specific commands and files in the first paragraph ?

Reply:
also, we didn't mention `dvc run` before

Reply:
OK, removed specific commands and dvc.yaml mentions.